src/handlers/comment/commands.ts

```diff
@@ -6,6 +6,7 @@ export enum IssueCommentCommands {
   PAYOUT = "/payout", // request permit payout
   MULTIPLIER = "/multiplier", // set bounty multiplier (for treasury)
   QUERY = "/query",
+  REVIEW = "/review", // GPT will review the pull request
```

Comment:
```suggestion
  REVIEW = "/review", // gpt-3.5-turbo-16k will review the pull request
```

Reply:
Why did you pick that model?

Reply:
Choose this one because, at least I do not have GPT-4 api access idk if that's a thing but I sure don't have access through my sub. 

This also has the fattest token limit out of the available models excluding the 32k models.

Reply:
It is double the price per token though but lets worry about cost optimization later.

Reply:
Yeah that's the fun part, tiktoken is the best token library for dealing with that but it's a python library. I found a ts wrapper for it but haven't used it